### ISSUE TITLE
QR Check In on Smart Devices

### ISSUE THREAD

**Issue by aethatsme** [Attendize/Attendize#126]
I tried on multiple mobile devices, the camera capture isn't triggered.

**Comment by daveearley** [comment]
I just checked an a previous commit is causing issues with recognizing QR codes. I'll push a fix shortly.

**Comment by aethatsme** [comment]
This issue still persist on version 1.10

**Comment by aethatsme** [comment]
I meant 1.0

**Comment by cosmorogers** [comment]
@aethatsme I believe you have to run the site over HTTPS for this to work, I've just been debugging it and got it to work without any code changes by running it over HTTPS

**Comment by aethatsme** [comment]
I have it running over HTTPS, same issue. Blue pop up screen will appear with the outline square but no camera triggered.

**Comment by cosmorogers** [comment]
Maybe it's the device you are using?
Do you get any console errors?
Try outputting the error passed to the function here
https://github.com/Attendize/Attendize/blob/master/public/assets/javascript/check_in.js#L125

**Comment by aethatsme** [comment]
Nope, I checked it on all my smart devices both android and ios. Same issue. 

See thread for screenshot. https://github.com/Attendize/Attendize/issues/195

Thinking about incorporating http://phonegap.com instead.
